_pages/cookbooks/file-system-to-marklogic.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: inner
+title: File System to MarkLogic
+lead_text: ''
+permalink: /file-system-to-marklogic
+---
+
+# File System to MarkLogic
+
+This example watches a directory for files, imports them into MarkLogic, then deletes them. The MarkLogic URI is /files/ followed by the filename. The NiFi filename property is set by the GetFile processor. 
+
+Many NiFi properties support embedded expressions, such as ${filename} in the ml.uri property below. For more about expressions, see [Apache NiFi Expression Language Guide.][nifi-exp-lang]
+
+
+<a href="./files/FileSystemToMarkLogic.xml" download>Download Template</a>
+
+## Processors:
+
+### GetFile
+
+reads files from a watched directory
+
+**Properties**
+
+Input Directory
+ : */some/path*
```

Comment:
Any fancy formatting required? For instance, on Windows, can you do stuff like "C:\Users\dcassel", or does the path need to be altered at all? (Probably a question for Frank.)

Reply:
You don't need to do any special formatting for the path from what I've seen. The original templates had a Windows path and they followed the standard format like "C:\Users\dcassel".